docker-compose.yml

```diff
@@ -122,6 +122,43 @@ services:
     labels:
       logs: "promtail"
 
+  hyperswitch-web:
+    ports:
+      - "9050:9050"
+      - "9060:9060"
+      - "5252:5252"
+    build:
+      context: ./docker
+      dockerfile: web.Dockerfile
+    networks:
+      - router_net
+    env_file:
+      - ./.env
+    environment:
+      - HYPERSWITCH_PUBLISHABLE_KEY=$HYPERSWITCH_PUBLISHABLE_KEY
+      - HYPERSWITCH_SECRET_KEY=$HYPERSWITCH_SECRET_KEY
+      - HYPERSWITCH_SERVER_URL=${HYPERSWITCH_SERVER_URL:-http://hyperswitch-server:8080}
+      - HYPERSWITCH_CLIENT_URL=${HYPERSWITCH_CLIENT_URL:-http://hyperswitch-web:9050}
+      - SELF_SERVER_URL=${SELF_SERVER_URL:-http://hyperswitch-web:5252}
+    depends_on:
+      hyperswitch-server:
+        condition: service_started
```

Comment:
@Dhanus3133 can you remove this since we want the option to run this along with local server.
i.e using the local cargo run along with docker services for hyperswitch-web & hyperswitch-control-center.